src/Xhgui/Saver.php

```diff
@@ -23,7 +28,20 @@ public static function factory($config)
                 $mongo = new MongoClient($config['db.host'], $config['db.options']);
                 $mongo->{$config['db.db']}->results->findOne();
                 $profiles = new Xhgui_Profiles($mongo->{$config['db.db']});
+		        // Get new profiling ID, if profiler is enabled
+		        if (call_user_func($config['profiler.enable'])) {
+			        self::$lastProfilingId = new MongoId();
```

Comment:
Ok. And as I moved the static method into the `Profiling.php` class, then there are no changes in `Saver.php` class. I reverted my changes in a separate commit:
https://github.com/perftools/xhgui-collector/pull/2/commits/c68ee2256b65b0685279757ab7d0b98b9494b760